README.md

```diff
@@ -125,7 +136,12 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+lamp = wishes.create(first_wish:"1" second_wish:"2", third_wish:"3", genie:'Genie')
+lamp.update(third_wish:"3")
+genie2 = 'Jafar'
+lamp2 = wishes.create(first_wish:"1",second_wish:"2", third_wish:"3", genie2:'Jafar')
+genie = 'nil'
+
```

Comment:
It looks like the Genie and the wishes are getting a bit mixed up here. I'd do something like:

```rb
genie = Genie.create(name: "Genie")
lamp = Lamp.create(wishes: 3, genie: genie)
lamp.wishes.update(wishes: 1)
jafar = Genie.create(name: "Jafar")
jafars_lamp = Lamp.create(wishes: 3, genie: jafar)
genie.lamp = nil
```